OpenRA.Mods.Common/Traits/Render/WithSpriteTurret.cs

```diff
@@ -124,7 +124,7 @@ public virtual void Tick(Actor self)
 			if (Info.AimSequence == null)
 				return;
 
-			var sequence = Attack.IsAttacking ? Info.AimSequence : Info.Sequence;
+			var sequence = Attack.Any(a => a.IsAttacking && !a.IsTraitDisabled) ? Info.Sequence : Info.AimSequence;
```

Comment:
https://github.com/OpenRA/OpenRA/pull/11600#discussion_r70274265
> Is the swapping of the sequences here intentional?

Here (not) as well?